tc-package.sh

```diff
@@ -10,11 +10,16 @@ tar -C ${HOME}/DeepSpeech/tf/bazel-bin/tensorflow/ \
 
 tar -C ${HOME}/DeepSpeech/tf/bazel-bin/native_client/ \
 	-uf /tmp/artifacts/native_client.tar \
-	libdeepspeech.so
+	libdeepspeech.so \
+	libdeepspeech_utils.so
 
 tar -C ${HOME}/DeepSpeech/ds/native_client/ \
 	-uf /tmp/artifacts/native_client.tar \
 	deepspeech
 
+if [ -s ${HOME}/DeepSpeech/ds/native_client/dist/* ]; then
```

Comment:
nit: man page for `-s` of `test` is not clear, does it ensure size greater than zero for *all* of the files, or just at least one of them ?

if possible I would prefer we `[ -d dist/ ]` and do a cp for each of the files.